.github/workflows/test_package.yml

```diff
@@ -0,0 +1,91 @@
+      - uses: actions/checkout@v2
+      - name: Prepare version name
+        id: prep
+        run: |
+          VERSION_TAG=edge
+          if [[ $GITHUB_REF == refs/tags/* ]]; then
+            VERSION_TAG=${GITHUB_REF#refs/tags/}
+          elif [[ $GITHUB_REF == refs/heads/* ]]; then
+            VERSION_TAG=-$(echo ${GITHUB_REF#refs/heads/} | sed -r 's#/+#-#g')
+          elif [[ $GITHUB_REF == refs/pull/* ]]; then
+            VERSION_TAG=-pr-${{ github.event.number }}
+          fi
+          if [ ${VERSION_TAG} != ${GITHUB_REF#refs/tags/} ]; then
+            VERSION_TAG=v${VERSION_TAG}-sha-${GITHUB_SHA::8}-SNAPSHOT
+          fi
+          echo ::set-output name=versionTag::${VERSION_TAG}
+      - name: Set up JDK 11
+        uses: actions/setup-java@v2
+        with:
+          java-version: '11'
+          distribution: 'adopt'
+          cache: maven
+      - name: Run Java tests
+        run: mvn test -Drevision=${{ steps.prep.outputs.versionTag }}
+      - name: Persist Java tests reports
+        if: always()
+        uses: actions/upload-artifact@v2
+        with:
+          name: Test reports
+          path: |
+            gtfs-realtime-validator-lib/target/surefire-reports
+            gtfs-realtime-validator-webapp/target/surefire-reports
+  package:
+    needs: [test]
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Prepare version name
+        id: prep
+        run: |
+          VERSION_TAG=edge
+          if [[ $GITHUB_REF == refs/tags/* ]]; then
+            VERSION_TAG=${GITHUB_REF#refs/tags/}
+          elif [[ $GITHUB_REF == refs/heads/* ]]; then
+            VERSION_TAG=-$(echo ${GITHUB_REF#refs/heads/} | sed -r 's#/+#-#g')
+          elif [[ $GITHUB_REF == refs/pull/* ]]; then
+            VERSION_TAG=-pr-${{ github.event.number }}
+          fi
+          if [ ${VERSION_TAG} != ${GITHUB_REF#refs/tags/} ]; then
+            VERSION_TAG=v${VERSION_TAG}-sha-${GITHUB_SHA::8}-SNAPSHOT
+          fi
+          echo ::set-output name=versionTag::${VERSION_TAG}
+      - name: Set up JDK 11
+        uses: actions/setup-java@v2
+        with:
+          java-version: '11'
+          distribution: 'adopt'
+          cache: maven
+      - name: Package lib/cli and web app jars with Maven
+        run: mvn -B package --file pom.xml -DskipTests -Drevision=${{ steps.prep.outputs.versionTag }}
+      - name: Persist lib jar
+        uses: actions/upload-artifact@v2
+        with:
+          name: gtfs-realtime-validator-lib-${{ steps.prep.outputs.versionTag }}.jar
+          path: gtfs-realtime-validator-lib/target/gtfs-realtime-validator-lib-${{ steps.prep.outputs.versionTag }}.jar
+      - name: Persist webapp jar
+        uses: actions/upload-artifact@v2
+        with:
+          name: gtfs-realtime-validator-webapp-${{ steps.prep.outputs.versionTag }}.jar
+          path: gtfs-realtime-validator-webapp/target/gtfs-realtime-validator-webapp-${{ steps.prep.outputs.versionTag }}.jar
```

Comment:
When I run the webapp downloaded from https://github.com/MobilityData/gtfs-realtime-validator/suites/5315339693/artifacts/165346975 using this command from the README (Windows 10, JDK 11.0.12), 

`java -Djsse.enableSNIExtension=false -jar gtfs-realtime-validator-webapp-v-pr-85-sha-1f238226-SNAPSHOT.jar`

I get:

![image](https://user-images.githubusercontent.com/928045/154161605-b47a6aa3-b7ef-46c0-8376-b3f85c8a324d.png)

Could you try running this on your machine and see if you get the same?

Reply:
Note - if I remove the `-Djsee...` configuration then it seems to run ok using the MBTA data from the README. At one point this was required for accessing GTFS or GTFS RT URLs served via HTTPS, but maybe not anymore?

The README says:

>If you're using `https` URLs for GTFS or GTFS-rt feeds, either:
>* Use the `-Djsse.enableSNIExtension=false` parameter when running the tool
>* Install the [Java Cryptography Extension (JCE)](https://www.oracle.com/technetwork/java/javase/downloads/jce8-download-2133166.html) - You will need to replace the `US_export_policy.jar` and `local_policy.jar` files in your JVM `/security` directory, such as `C:\Program Files\Java\jdk1.8.0_73\jre\lib\security`, with the JAR files in the JCE Extension download.

I haven't installed JCE on this machine separately from the main JDK.

Reply:
It's possible the above issue was specific only to Java 8. 

@maximearmstrong If you have the same experience of being able to run on JDK 11 without `-Djsse.enableSNIExtension=false` or installing JCE, and the CI is running on JDK 11, I suggest we update the docs to suggest using JDK 11 instead of JDK 8 (which is a good best practice anyway given the age of Java 8). This also means we'd remove the above docs related to HTTPS.

Reply:
Good observation! Indeed, it is running perfectly on my side with JDK 11 and without -Djsse.enableSNIExtension=false. I will update the readme.